python/paddle/fluid/tests/unittests/test_jit_save_load.py

```diff
@@ -187,6 +187,31 @@ def forward(self, x, y):
         return x + y
 
 
+class LinearNetWithMultiStaticFunc(fluid.dygraph.Layer):
+    def __init__(self, in_size, out_size):
+        super(LinearNetWithMultiStaticFunc, self).__init__()
+        self._linear_0 = Linear(in_size, out_size)
+        self._linear_1 = Linear(in_size, out_size)
+        self._scale = paddle.to_tensor(9.9)
+
+    @declarative
```

Comment:
better to use `paddle.jit.to_static`, `declarative` will be removed later, we better not add this writing

Reply:
Done, thanks.